examples/host_vars/node1.yml

```diff
@@ -52,3 +52,10 @@ hugepages_2M: 0
 # CPU isolation from Linux scheduler
 isolcpus_enabled: true
 isolcpus: "4-7"
+
+# Intel CommsPowerManagement
+sst_bf_enabled: true
+# [s] Set SST-BF config (set min/max to 2700/2700 and 2100/2100)
+# [m] Set P1 on all cores (set min/max to 2300/2300)
+# [r] Revert cores to min/Turbo (set min/max to 800/3900)
+# sst_bf_mode: {s,m,r}
```

Comment:
please set `s` option as a default and uncommented option, i.e.
`sst_bf_mode: s`

Reply:
add a comment that this setting requires `sst_bf_configuration_enabled` to be set as `true`

Reply:
also, not a fan of those letters,
wonder if we could replace it with:
s -> sst_bf or enabled
m -> all_p1state
r -> min_turbo or disabled
would be good to ask SST-BF folks for their opinion